### PR TITLE
Fix problems in #1819

### LINKED ISSUE #1819: `sea-orm-cli` entities generator panics when an item in EnumType consist entirely without any letters or numbers
## Description

When I use this enum type in MySQL:

```sql
ENUM('Question','QuestionsAdditional','Answer','Other','/')
```

The `sea-orm-cli` entities generator will panic because it cannot convert '/' into an enum item name in Rust. That's just fine, but what troubles me is the `sea-orm-cli` didn't give a clear explanation for this panic. It just shows this:

```
thread 'main' panicked at 'Ident is not allowed to be empty; use Option<Ident>', <HomePath>\.cargo\registry\src\mirrors.ustc.edu.cn-61ef6e0cd06fb9b8\proc-macro2-1.0.66\src\fallback.rs:752:9
```

this is not really a bug itself, but we could probably provide users with a clearer error message or do some extra handling to accommodate these values. And if you'd be willing to first introduce a friendly error message as a transitional step before proposing other better approaches later, I'd be happy to provide a pull request for that.

Code related are mainly in this file:`sea-orm-codegen/src/entity/active_enum.rs:23`

## Steps to Reproduce

1. Setup a MySQL server, and run this:
 ```sql
CREATE TABLE `test` (
  `enum_column` enum('A','B','/') NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4
 ```
2. Try to use `sea-orm-cli` to generate entity

### Expected Behavior

A friendly panic message is given, maybe saying 'We currently do not support enum items consisting entirely without any letters or numbers.', or just process this situation fine.

### Actual Behavior

A confusing error message unrelated to the specific cause was given.

### Reproduces How Often

Always 

### Workarounds

this is fine :)
 ```sql
CREATE TABLE `test` (
  `enum_column` enum('A','B','/') NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4
 ```

## Versions

image

### PR DESCRIPTION
**Pull request by Cinea4678**
## PR Info

- Closes #1819 

- Dependencies:
  - 

- Dependents:
  - 

## New Features

- [ ] 

## Bug Fixes

- [x] `sea-orm-cli` entities generator panics when an item in EnumType consist entirely without any letters or numbers

## Breaking Changes

- [ ] 

## Changes

- [ ]

### CODE REVIEW

File: sea-orm-codegen/src/entity/active_enum.rs [hunk 1]
```diff
@@ -20,7 +20,18 @@ impl ActiveEnum {
             if v.chars().next().map(char::is_numeric).unwrap_or(false) {
                 format_ident!("_{}", v)
             } else {
-                format_ident!("{}", v.to_upper_camel_case())
+                let variant_name = v.to_upper_camel_case();
+                if variant_name.is_empty() {
+                    println!("Warning: item '{}' in the enumeration '{}' cannot be converted into a valid Rust enum member name. It will be converted to its corresponding UTF-8 encoding. You can modify it later as needed.", v, enum_name);
```
Comment (tyt2y3) [inline sea-orm-codegen/src/entity/active_enum.rs:25]: @billy1624 is println or log::warn the way to go?

**Comment by tyt2y3** [comment]
Thank you for the contribution. Looks good to me, but would require Billy's review as well.

**Comment by pauliyobo** [comment]
Hello.
Would it be possible to know the status of this PR?

**Comment by tyt2y3** [comment]
Sorry for the delay. I think @billy1624 will review this soon

**Pull request merged**